src/main/java/TaskManager.java

```diff
@@ -0,0 +1,60 @@
+public class TaskManager {
+    public static final int MAX_ENTRIES = 100;
+    int numberOfItems = 0;
+    Task[] itemList = new Task[MAX_ENTRIES];
+
+    public void handleCommand(String command) {
+         if (command.equals("list")) {
+             printList();
+         }
+        else if (command.startsWith("mark") || command.startsWith(("unmark"))) {
+            handleMarkAsDone(command);
+        }
+        else {
+            handleAddTask(command);
+        }
+    }
+
+    private void printList() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < numberOfItems; i++) {
+            System.out.println(i + 1 + "." + itemList[i].toString());
```

Comment:
Could this code be made more explicit and obvious, by using parentheses for the index to group it separately from the itemList string being printed?